net/scripts/gce-provider.sh

```diff
@@ -182,7 +182,7 @@ cloud_CreateInstances() {
     imageName="ubuntu-2004-focal-v20201211-with-cuda-10-2 --image-project principal-lane-200702"
   else
     # Upstream Ubuntu 20.04 LTS image
-    imageName="ubuntu-2004-focal-v20201201 --image-project ubuntu-os-cloud"
+    imageName="ubuntu-2004-focal-v20220419 --image-project ubuntu-os-cloud"
```

Comment:
i'll ask again. to what end?

at the very least this is unrelated to the rest of the change set and deserves its own PR.  don't worry, we can afford the github issue numbers :wink:

Reply:
why -- because kernel version for image v20201201 doesn't have dpkg containing `perf` in the dpkg artifactory. So without updating kernel version it is impossible to install perf.

I created a spin off PR: https://github.com/solana-labs/solana/pull/25690/files